script/e2e-test.sh

```diff
@@ -432,12 +410,39 @@ admin_token="$(kubectl get -n kubeapps secret "$(kubectl get -n kubeapps service
 view_token="$(kubectl get -n kubeapps secret "$(kubectl get -n kubeapps serviceaccount kubeapps-view -o jsonpath='{.secrets[].name}')" -o go-template='{{.data.token | base64decode}}' && echo)"
 edit_token="$(kubectl get -n kubeapps secret "$(kubectl get -n kubeapps serviceaccount kubeapps-edit -o jsonpath='{.secrets[].name}')" -o go-template='{{.data.token | base64decode}}' && echo)"
 
-## Run tests
-info "Running Integration tests..."
+info "Running main Integration tests without k8s API access..."
 if ! kubectl exec -it "$pod" -- /bin/sh -c "INTEGRATION_RETRY_ATTEMPTS=3 INTEGRATION_ENTRYPOINT=http://kubeapps-ci.kubeapps USE_MULTICLUSTER_OIDC_ENV=${USE_MULTICLUSTER_OIDC_ENV} ADMIN_TOKEN=${admin_token} VIEW_TOKEN=${view_token} EDIT_TOKEN=${edit_token} yarn start ${ignoreFlag}"; then
   ## Integration tests failed, get report screenshot
   warn "PODS status on failure"
   kubectl cp "${pod}:/app/reports" ./reports
   exit 1
 fi
-info "Integration tests succeeded!!"
+info "Main integration tests succeeded!!"
+
+## Upgrade and run operator test
+# Operators are not supported in GKE 1.14 and flaky in 1.15, skipping test
+if [[ -z "${GKE_BRANCH-}" ]] && [[ -n "${TEST_OPERATORS-}" ]]; then
+  installOLM "${OLM_VERSION}"
+
+  # Update Kubeapps settings to enable operators and hence proxying
+  # to k8s API server.
+  info "Installing latest Kubeapps chart available"
+  installOrUpgradeKubeapps bitnami/kubeapps \
+    "--set" "featureFlags.operators=true"
```

Comment:
That is a brilliant idea Michael!
Kubeapps resources are recreated to enable operators without the need to deploy the whole cluster again.